lib/bots/procbot.ts

```diff
@@ -130,9 +130,9 @@ export class ProcBot<T> {
         // Swap out known tags that become booleans.
         const minimumVersion = ((config || {}).procbot || {}).minimum_version;
         if (minimumVersion && process.env.npm_package_version) {
-            config.procbot.minimum_version = (process.env.npm_package_version < minimumVersion) ? false : true;
-        } else {
-            config.procbot.minimum_version = true;
+            if (process.env.npm_package_version < minimumVersion) {
+                throw new Error('Current ProcBot implementation does not minimum required version for the operations');
```

Comment:
...but I still think you ought to change the message from:
`'Current ProcBot implementation does not minimum required version for the operations'`
to
`'Current ProcBot implementation does not meet minimum required version for the operations'`

(i.e. insert the missing `meet` word)